odh-dashboard/modelserving/caikit-ootb.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/ootb: 'true'
+  annotations:
+    description: Caikit is an AI toolkit that enables users to manage models through a set of developer friendly APIs. It provides a consistent format for creating and using AI models against a wide variety of data domains and tasks.
+    openshift.io/provider-display-name: Red Hat, Inc.
+    tags: rhods,rhoai,kserve,servingruntime
+    template.openshift.io/documentation-url: https://github.com/opendatahub-io/caikit-tgis-serving
+    template.openshift.io/long-description: This template defines resources needed to deploy caikit-tgis-serving servingruntime with Red Hat Data Science KServe for LLM model
+    template.openshift.io/support-url: https://access.redhat.com
+    opendatahub.io/modelServingSupport: '["single"]'
+  name: caikit-tgis-serving-template
+objects:
+- apiVersion: serving.kserve.io/v1alpha1
+  kind: ServingRuntime
+  metadata:
+    name: caikit-tgis-runtime
+    annotations:
+      openshift.io/display-name: Caikit TGIS ServingRuntime for KServe
+  labels:
+    opendatahub.io/dashboard: 'true'
+  spec:
+    multiModel: false
+    supportedModelFormats:
+      - autoSelect: true
+        name: caikit
+    containers:
+      - name: kserve-container
+        image: text-generation-inference
```

Comment:
quay.io/opendatahub/text-generation-inference:fast

Reply:
These have always been environment variables passed to Kustomize... @lucferbux are these handled?

Reply:
Yes, this is kustomization image replacements, the sha is in the bas kustomize file in the PR.

Reply:
ok sorry then :)